massa-bootstrap/src/server_binder.rs

```diff
@@ -148,7 +148,7 @@ impl<D: Duplex> BootstrapServerBinder<D> {
         )
         .map_err(|e| match e {
             BootstrapError::IoError(e) if e.kind() == ErrorKind::WouldBlock => {
-                BootstrapError::IoError(ErrorKind::TimedOut.into())
+                BootstrapError::TimedOut(ErrorKind::TimedOut.into())
```

Comment:
I think we can use the `impl From<std::io::Error> for BootstrapError` here, no?

Reply:
hm don't think , `e` is now `BootstrapError` and not `IoError`